src/payment/payment-request-options.ts

```diff
@@ -168,4 +169,10 @@ export interface PaymentInitializeOptions extends PaymentRequestOptions {
      * They can be omitted unless you need to support StripeV3.
      */
     stripev3?: StripeV3PaymentInitializeOptions;
+
```

Comment:
StripeV3?

Reply:
oh no, i added the mollie option down here

Reply:
Look at the documentation.

Reply:
my bad, now is fix